shared/teams/add-members-wizard/confirm.tsx

```diff
@@ -66,7 +67,7 @@ const AddMembersConfirm = () => {
         addMembers(
           [
             {
-              defaultChannelsOverride: defaultChannels
+              addToChannels: defaultChannels
```

Comment:
shouldn't we filter out the defaults to only send the non-defaults here?

Reply:
this component doesn't know what the team wide default channels are, these are actually the additional channels. this `defaultChannels` should be renamed actually, will do